x-pack/test/api_integration/apis/search/session.ts

```diff
@@ -189,7 +188,7 @@ export default function ({ getService }: FtrProviderContext) {
 
         const { id: id2 } = searchRes2.body;
 
-        await retry.waitForWithTimeout('searches persisted into session', 5000, async () => {
+        await retry.waitFor('searches persisted into session', async () => {
```

Comment:
In some places just changing `waitForWithTimeout` -> `waitFor` to keep it simpler,
leaving `waitForWithTimeout` only where we for sure know that default `waitFor` timeout is not long enough